### PR TITLE
API-88 Document how to get client credentials

### PR DESCRIPTION
**Pull request by bertramakers**
### Added

- Introduction page
- How to get client credentials

---

Ticket: https://jira.uitdatabank.be/browse/API-88

Note: This PR only focuses on the scope described above. Other improvements to the authentication documentation are coming in different tickets / PRs.

Preview: https://docs.publiq.be/docs/authentication/branches/authentication%2FAPI-88-getting-started/d6a8ba3f3c186-introduction (login with apidocs@publiq.be user required)

### CODE REVIEW

File: projects/authentication/docs/introduction.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Introduction
+
+This space gives an overview of the possible authentication methods on publiq's APIs.
+
+In the past, almost every one of our APIs used a different authentication method. To provide a consistent experience and reduce the work needed to integrate with multiple of our APIs, we have evolved our APIs to support a shared authentication approach.
+
+To start, you will need a set of client credentials consisting of a **client id** and **client secret** which we will provide you. You can then use these in 3 ways to authenticate your requests to our APIs, depending on the API(s) you integrate with and your function use case. See the [authentication methods overview](./methods.md) for more info about the various ways that you can authenticate your requests.
```
Comment (JurgenG) [inline projects/authentication/docs/introduction.md:7]: ... and your **function** use case. ...

I have the feeling that the word "function" is redundant. Could it be left away?

File: projects/authentication/docs/requesting-credentials.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+## UiTPAS
+
+To use [UiTPAS API v4](https://docs.publiq.be/docs/uitpas), see its [quickstart guide](https://docs.publiq.be/docs/uitpas/d0748f47a3dba-quick-start) to request a client id and client secret via the linked Google Form.
+
+## UiTdatabank
+
+UiTdatabank APIs currently uses a custom authentication method based on API keys instead of client ids and secrets.
+
+While support for the standardized authentication methods documented in this space has been implemented, new client ids and secrets are only provided to a select few partners that are trying out these new authentication methods in UiTdatabank. In the future this will be opened up to all integrators.
+
```
Comment (JurgenG) [inline projects/authentication/docs/requesting-credentials.md:16]: People who want to combine UiTPAS and Entry API, can follow the flow described in the UiTPAS quickstart guide.

As a sidenote: ledenbeheer.be has customers who have UiTPAS (they need both UPS and Entry), other clients aren't in UiTPAS yet (they only need Entry). Obviously they follow the same flow for all requests.

File: projects/authentication/docs/requesting-credentials.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+1.  Contact us at vragen@uitdatabank.be to get a client id and client secret to start using the new authentication mechanism already. Once you have your client id and secret you can use the standard authentication methods described here.
+2.  **Or**, register your project at [Projectaanvraag](https://projectaanvraag.uitdatabank.be) to automatically get a test **API key** that you can use as described on the [EntryAPI authentication documentation](https://documentatie.uitdatabank.be/content/entry_api\_3/latest/authentication.html) in our older documentation portal. While this way of authentication will be phased out in the future for new integrators, it will still be supported for existing integrations for the foreseeable future.
+
+Aside from the authentication method all API operations work exactly the same whether you have a client id and secret, or an API key.
+
+## museumPASSmusées
+
+Access to the [museumPASSmusées Partner API](https://docs.publiq.be/docs/museumpassmusees) is only provided to very specific partners at the moment as the functionality is very specific to museum visits. To request your credentials, please contact support@museumpassmusees.be.
```
Comment (JurgenG) [inline projects/authentication/docs/requesting-credentials.md:26]: 2x "very specific" ... maybe replace by:
... as the functionality is tailored to museum visits.

**Review by JurgenG — CHANGES_REQUESTED** [review]
A few small proposals to improve.

Otherwise, nice and clear text.

File: projects/authentication/docs/introduction.md (hunk 1 above)
Comment (bertramakers) [reply projects/authentication/docs/introduction.md:7]: Was meant to be _functional_ use case 🙈  But maybe it's also fine without it yes!

File: projects/authentication/docs/requesting-credentials.md (hunk 2 above)
Comment (bertramakers) [reply projects/authentication/docs/requesting-credentials.md:16]: I will mention it for UiTPAS integrators. But I'm not sure if clients that do not need UiTPAS should already start using client id & secret instead of the API key flow. Gezinsbond did it to start using client access tokens, but it had the unintended side-effect that their users could could not edit the events/places created with the client access tokens in UDB3 since then, while they expected that to keep working. Plus it could mean a lot of extra manual work to set up their clients.

File: projects/authentication/docs/requesting-credentials.md (hunk 3 above)
Comment (bertramakers) [reply projects/authentication/docs/requesting-credentials.md:26]: I will reword it to "... is only provided to very specific partners for now as the primary use case at the moment is registering museum visits via a card reader", if okay with you. That maybe better explains the exclusivity.

**Review by JurgenG — APPROVED** [review]
